members.ts

```diff
@@ -220,5 +220,16 @@ export const members: Member[] = [
     twitterUsername: 'kensan_p',
     //githubUsername: '',
     websiteUrl: 'https://zenn.dev/kenshiro',
+  },
+  {
+    id: 'ume3',
+    nickname: 'ume3',
+    realName: '梅谷敦',
+    bio: 'EM',
+    avatarSrc: '/avatars/bob.jpg',
```

Comment:
ファイル名、追加しているものと異なっているので調整お願いします

Reply:
す、すみません！直前で差し替えたのでした。命名変更します。
```suggestion
    avatarSrc: '/avatars/ume3.jpg',
```